src/domain/occurrence/CreateOccurrencePage.tsx

```diff
@@ -135,16 +158,31 @@ const CreateOccurrencePage: React.FC = () => {
     }
   };
 
+  const handleDeleteOccurrence = async (
+    occurrence: OccurrenceFieldsFragment
+  ) => {
+    try {
+      await deleteOccurrence({ variables: { input: { id: occurrence.id } } });
+      refetchEvent();
+    } catch (e) {
+      toast(t('occurrences.deleteError'), {
+        type: toast.TYPE.ERROR,
+      });
```

Comment:
Sorry, I missed this at first time. Maybe could use `toast.error(t('occurrences.deleteError'))` also here